File: php/update.php
<?php
$m = new MongoClient();
echo "Connection to database successfully"; 
$db = $m -> MYDB259743
echo "\n Database selected successfully";
$col = $db -> MyCol;
echo "\n Connection selected successfully";""
$col-> update(array ("name" => "Chanchal"),array('$set'=>array("age" => 19 ) ) ) ;
echo "\n Document updated successfully";
?>

File: python/update.py
from pymongo import MongoClient
client=MongoClient('localhost',27017)
db=client.TYITDB259743
def update():
    try:
        name1=input("Enter the Name: ")
        age1=input("Enter the Age to update: ")
        db.MyCol.update_one({"name":name1},{"$set":{"age":age1}})
        print("\nRecords update successfully\n")
    except Exception:
        print(str(e))

update()
